apps/desktop-wallet/src/contexts/wallet.tsx

```diff
@@ -16,53 +16,27 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { Wallet } from '@alephium/shared-crypto'
 import { createContext, useContext, useState } from 'react'
 
 import { CenteredSection } from '@/components/PageComponents/PageContainers'
 
 export interface WalletContextType {
-  plainWallet?: Wallet | undefined
-  setPlainWallet: (w: Wallet | undefined) => void
-  mnemonic: string
-  setMnemonic: (m: string) => void
-  walletName: string
-  setWalletName: (w: string) => void
-  password: string
-  setPassword: (password: string) => void
+  mnemonic: Buffer | null
+  setMnemonic: (mnemonic: Buffer | null) => void
```

Comment:
The wallet creation flow is the only place we are caching the mnemonic in the desktop wallet. This is because we have multiple screens to see the mnemonic, verify it, and give a password to encrypt and store it.

Reply:
Side note for other potential reviewers: when we studied other wallets, the mnemonic was inspectable in memory during wallet creation in all of them. This seems to be inevitable.